src/mig/agent/agent.go

```diff
@@ -163,8 +163,8 @@ func runModuleDirectly(mode string, args []byte, pretty bool) (out string) {
 		return fmt.Sprintf(`{"errors": ["module '%s' is not available"]}`, mode)
 	}
 	// instanciate and call module
-	modRunner := modules.Available[mode].Runner()
-	out = modRunner.(modules.Moduler).Run(os.Stdin)
+	run := modules.Available[mode].NewRun()
```

Comment:
scratch that, it's implemented in the `modules.Moduler` interface.
(really wish github could stage comments...)